src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies logging level filtering works
```powershell
Expected $null, but got PSLogItem.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 256
256:         $items | Should -Be $null
```
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```